presto-native-execution/presto_cpp/main/tests/TaskManagerTest.cpp

```diff
@@ -1132,6 +1138,71 @@ TEST_F(TaskManagerTest, checkBatchSplits) {
   ASSERT_EQ(resultOrFailure.status, nullptr);
 }
 
+TEST_F(TaskManagerTest, buildSpillDirectryFailure) {
+  // Cleanup old tasks between test iterations.
+  taskManager_->setOldTaskCleanUpMs(0);
+  for (bool buildSpillDirectoryFailure : {false}) {
+    SCOPED_TRACE(fmt::format(
+        "buildSpillDirectoryFailure: {}", buildSpillDirectoryFailure));
+    auto spillDir = setupSpillPath();
+
+    auto filePaths = makeFilePaths(1);
```

Comment:
Why do we need to write data to files? Can we not use values(batches) node in the plan?